test/utils/test_utils.dart

```diff
@@ -7,18 +7,18 @@ import 'package:flutter_adaptive_cards/flutter_adaptive_cards.dart';
 class MyTestHttpOverrides extends HttpOverrides {}
 
 Widget getWidthDefaultHostConfig(String name) {
-  return getWidget(name, 'host_config');
+  return getWidget(name, 'host_config.json');
 }
 
 Map? getDefaultHostConfig() {
-  var hostConfigFile = File('host_configs/host_config');
+  var hostConfigFile = File('host_configs/host_config.json');
   var config = hostConfigFile.readAsStringSync();
   return json.decode(config);
 }
 
 Widget getWidget(String path, String hostConfigPath) {
-  var file = File('test/samples/$path');
-  var hostConfigFile = File('test/host_configs/$hostConfigPath');
+  var file = File('test/samples/$path.json');
+  var hostConfigFile = File('test/host_configs/$hostConfigPath.json');
```

Comment:
```suggestion
  var hostConfigFile = File('test/host_configs/$hostConfigPath');
```

I could not find an error in the examples and am not sure where it is used there, but at first sight it seems wrong.

It get's called on line 10 and in the `sample_some_tests.dart` on line 17 with the `.json` extension already set.

Reply:
good catch, changed it